zerver/openapi/zulip.yaml

```diff
@@ -21737,6 +21776,16 @@ components:
             Depending on the size of the organization, it can take
             anywhere from seconds to an hour to generate the data
             export.
+        export_type:
+          type: integer
+          description: |
+            Whether the data export is a public data export or a
+            full data export with member consent.
+
+            - 1 = Public data export.
+            - 2 = Full data export with member consent.
+
+            **Changes**: New in Zulip 10.0 (feature level 300).
```

Comment:
This changes entry should continue with "Previously, only public data exports were supported via this API endpoint."

Reply:
Oh, actually, this is a case where it should be "Previously, the export type was not included in these objects. While only public data exports could be created via the API or UI, it was possible for other exports done by server administrators to be listed here."

Reply:
Added:
> Previously, the export type was not included in these objects because only public data exports could be created or listed via the API or UI.

This is not correct, right ?
> it was possible for other exports done by server administrators to be listed here.

Reply:
I think we had a small number that appeared there across all of Zulip Cloud, but it is probably irrelevant as a detail, so maybe not worth mentioning.